src/sentry/runner/initializer.py

```diff
@@ -67,6 +67,7 @@ def get_asset_version(settings):
     'system.databases': 'DATABASES',
     'system.debug': 'DEBUG',
     'system.secret-key': 'SECRET_KEY',
+    'system.rate-limit': 'SENTRY_SYSTEM_MAX_EVENTS_PER_MINUTE',
```

Comment:
This shouldn't be done here.

Ideally, you'd just change every use of `SENTRY_SYSTEM_MAX_EVENTS_PER_MINUTE` instead to use the new options.

Then add an entry into `apply_legacy_settings` to move existing settings.SENTRY_SYSTEM_MAX_EVENTS_PER_MINUTE` into `SENTRY_OPTIONS`.

If we do it like this, the value won't change at runtime since it gets memoized into the settings value.